.drone.yml

```diff
@@ -87,6 +87,18 @@ pipeline:
     commands:
       - make test-php-integration-dbg
 
+  cli-acceptance-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    environment:
+    - TEST_SERVER_URL=http://owncloud
+    - PLATFORM=Linux
+    - BEHAT_SUITE=${BEHAT_SUITE}
+    commands:
+    - make test-acceptance-cli
```

Comment:
I don't see any ``test-acceptance-cli`` target in ``Makefile`` - so I think you need to add that also in ``Makefile``